src/Codeception/Lib/Driver/Facebook.php

```diff
@@ -15,93 +18,98 @@ public function __construct($config, $logCallback = null)
             $this->logCallback = $logCallback;
         }
 
-        parent::__construct($config); // TODO: Change the autogenerated stub
+        FacebookSession::setDefaultApplication($config['app_id'], $config['secret']);
+        $this->appSession = FacebookSession::newAppSession($config['app_id'], $config['secret']);
+        $this->appId = $config['app_id'];
     }
 
     /**
-     * @inheritdoc
+     * @param $name
+     * @param array $permissions
+     *
+     * @return array
      */
-    protected function setPersistentData($key, $value)
+    public function createTestUser($name, array $permissions)
```

Comment:
If the name is optional then it should be passed as optional param as well, I think.